system-tests-new/zero_config/test_zero_config.py

```diff
@@ -0,0 +1,23 @@
+from pathlib import Path
+
+from fab.cli import _generic_build_config
+
+
+class TestZeroConfig(object):
+
+    def test_fortran_dependencies(self, tmp_path):
+        # test the sample project in the fortran dependencies system test
+        config = _generic_build_config(
+            Path(__file__).parent.parent / 'FortranDependencies',
+            kwargs={'fab_workspace': tmp_path, 'multiprocessing': False})
+        config.run()
+        assert (config.project_workspace / 'first.exe').exists()
+        assert (config.project_workspace / 'second.exe').exists()
```

Comment:
Is it enough to test for the existence of an executable or do we need to run it as well?

Reply:
I think we should run all the exes we produce in our system tests, but I'm not sure it's a high priority yet.